doc/notebooks/GSoC2021_GRASS_Jupyter_Integration.ipynb

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
The name can be just `jupyter_integration` or `jupyter` or some other simple name.